src/search_index.rs

```diff
@@ -215,6 +215,17 @@ impl SearchIndex {
                         Ok(content) => content,
                         Err(_) => return, // Skip files we can't read as text
                     };
+
+                    // Skip files with lines longer than 8096 bytes
+                    const MAX_LINE_LENGTH: usize = 8096;
+                    if content.lines().any(|line| line.len() > MAX_LINE_LENGTH) {
+                        eprintln!(
+                            "Skipping {}: contains line(s) longer than {} bytes",
+                            file_path.display(),
+                            MAX_LINE_LENGTH
+                        );
```

Comment:
Drop the log, we need to introduce more user friendly logging that can be disabled, but until we have done that we shouldn't spam the user